src/index.js

```diff
@@ -95,25 +181,26 @@ function handlebars(options) {
         template = template.code;
       }
 
-      const escapePath = (path) => path.replace(/\\/g, '\\\\');
+      const escapePath = (path) => jsesc(path, { minimal: true, wrap: true });
 
-      let body = `import Handlebars from '${escapePath(options.handlebars.id)}';\n`;
-      if (options.jquery) body += `import $ from '${escapePath(options.jquery)}';\n`;
+      let body = `import Handlebars from ${escapePath(options.handlebars.id)};\n`;
+      if (jqueryPath) body += `import $ from ${escapePath(jqueryPath)};\n`;
+      body += `import { getFormat, parsers } from ${escapePath(PLUGIN_HELPER_MODULE_ID)};\n`;
 
       if (options.helpers) {
         // Support `helpers` being singular or plural.
-        [].concat(options.helpers).forEach((helpers, i) => {
-          body += `import Helpers${i} from '${escapePath(helpers)}';\n`;
+        for (const [i, helpers] of [].concat(options.helpers).entries()) {
+          body += `import Helpers${i} from ${escapePath(helpers)};\n`;
           body += `if (!Helpers${i}.__initialized) {\n`;
           body += `  Helpers${i}(Handlebars);\n`;
           body += `  Helpers${i}.__initialized = true;\n`;
           body += `}\n`;
-        });
+        }
       }
 
       for (const partial of scanner.partials) {
         // Register the partial dependencies as partials.
-        body += `import '${escapePath(partial)}${options.templateExtension}';\n`;
+        body += `import ${escapePath(partial)}${options.templateExtension};\n`;
```

Comment:
Unless I'm misunderstanding what `options.templateExtension` is, I think the single quote at the end of `escapePath(partial)` needs to be moved after `${options.templateExtension}`.

Reply:
Yep, good call.